arsceneview/src/main/java/io/github/sceneview/ar/ArSceneView.kt

```diff
@@ -417,38 +416,32 @@ open class ArSceneView @JvmOverloads constructor(
         }
     }
 
-    override fun onTouch(selectedNode: Node?, motionEvent: MotionEvent): Boolean {
-        // TODO : Should be handled by the nodesTouchEventDispatcher
-        //nodeGestureRecognizer.onNodeTap(null)
+    override fun onTap(node: Node?, renderable: Renderable, motionEvent: MotionEvent) {
+        super.onTap(node, renderable, motionEvent)
 
-        arSession?.let { session ->
-            session.currentFrame?.hitTest(motionEvent)?.let { hitResult ->
-                onTouchAr(hitResult, motionEvent)
-                return true
+        gestureDetector.selectedNode = node as? ArNode
```

Comment:
I also made it possible to deselect the current node by taping on an empty space. I think I should restore the old behavior when it is only possible to select another node.

Reply:
My opinion is that taping an empty space should deselect the node

Reply:
+1 Maybe we could have it in a way that makes it easy for devs to restore the old behavior. Like having a flag to set or overwrite a method with an empty implementation.